src/annotator/config/index.js

```diff
@@ -162,9 +198,35 @@ export function getConfig(appContext = 'annotator', window_ = window) {
   let filteredKeys = configurationKeys(appContext);
   filteredKeys.forEach(name => {
     const configDef = configDefinitions[name];
+    const hasDefault = configDef.defaultValue !== undefined; // A default could be null
+    const browserExtensionTrue = isBrowserExtension(
```

Comment:
The name `browserExtensionTrue` is a little awkward. I see this was done to avoid a conflict with the `isBrowserExtension` function. I think it might have been better to rename that function (eg. `isURLFromBrowserExtension`)